pkg/controller/provider/validation.go

```diff
@@ -362,6 +360,21 @@ func (r *Reconciler) inventoryCreated(provider *api.Provider) error {
 	return nil
 }
 
+func (r *Reconciler) handleServerCreationFailure(provider *api.Provider, err error) {
+	provider.Status.Phase = ConnectionAuthFailed
+	msg := fmt.Sprint("The OVA provider server creation failed -", err)
```

Comment:
Do we want to leave the error msg at disply or only log it?

Reply:
if it looks alright (the error message) then I don't see why not to show it

Reply:
maybe just for consistency:
```suggestion
	msg := fmt.Sprint("The OVA provider server creation failed: ", err)
```